src/main/java/uk/gov/hmcts/darts/retention/model/CaseRetentionConfidenceReason.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.darts.retention.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+import lombok.extern.jackson.Jacksonized;
+
+import java.util.List;
+
+@Data
+@Builder
+@Jacksonized
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CaseRetentionConfidenceReason {
```

Comment:
Should this be hooked up to anything?

Reply:
No this ticket is just to create the json object and prove it can be converted to json